rest-server/engines/src/main/scala/com/actionml/engines/ur/UREngine.scala

```diff
@@ -141,7 +141,9 @@ object UREngine extends JsonSupport {
       mirrorType: Option[String] = None,
       mirrorContainer: Option[String] = None,
       sharedDBName: Option[String] = None,
-      sparkConf: Map[String, JValue])
+      sparkConf: Map[String, JValue],
+      algorithm: URAlgorithmParams,
```

Comment:
I thought about this and URAlgorithmParams is a requirement for this Engine. So this is correct